### ISSUE TITLE
Join items from different data sources

### ISSUE THREAD

**Issue by alastair** [trompamusic/ce-musicbrainz-import#7]
We need to make sure that we have sameas relationships or similar between the same data from different sources, e.g. musicbrainz/wikipedia/imslp

Where relations exist, we should import these. e.g. if musicbrainz links to wikidata and imslp, add these.
If there are no relations we need to do manual matching. I'm not sure of the best way to do this. For now we could use basic string matching / close matches. If our dataset is small enough, this could be done manually. If we create relations that don't previously exist, we should add them to musicbrainz

**Comment by musicog** [comment]
Importing existing relationship should be first priority.

For new relationships, fuzzy string matching is a reasonable first step but is also perilous -- rather different spellings across datasets are common particularly with early music (consider e.g. Orlando / Roland), as is shared names across distinct composers (pathological example: the family Ferrabosco, with three Alfonsos, along with a Domenico, Henry and John, all of whom were composers and musicians). At a minimum, we should use an authority service like viaf.org to guide matching activity. 

I would be remiss not to mention a paper we wrote on this subject at Oxford: https://link.springer.com/article/10.1007/s00799-017-0223-9#Fn12

This describes a Semantic Alignment and Linking Tool we implemented, which generates alignment suggestions based on both string similarity and semantic overlap (e.g. where birth / death dates, or titles of compositions, or other bits of context are available in multiple unaligned datasets), that then can be vetted relatively quickly by a domain expert for potential interlinking. 

Not proposing necessarily that we need to employ this within TROMPA -- particularly for the "big name" composers, manual interlinking may suffice. Something we could potentially come back to for interlinking of long-tail entities though.
